theme/styles/_global.sass

```diff
@@ -0,0 +1,66 @@
+/*! Default styles from your beloved manGoweb (http://mangoweb.cz) */
+
+*, *:before, *:after
+	box-sizing: border-box
+
+html
+	font-size: $base-font-size
```

Comment:
might be better like this:

```sass
*, *:before, *:after
  box-sizing: inherit

html
  box-sizing: border-box
```

What do we think? @enzy @Onset

Reply:
Looks good to me. 👍